packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -673,6 +675,24 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
     }
 }
 
+function ensureImageHasSpanParent(image: HTMLImageElement) {
```

Comment:
Let's move this function into setDOMSelection.ts and call it there.

Because SelectionPlugin is not the only entry point to select image, it can also be selected by code.